### PR TITLE
Homogenize examples' output.tf

### PR DESCRIPTION
**Pull request by fsismondi**

### CODE REVIEW

**Review by raphink — APPROVED** [review]

File: examples/eks-aws/terraform/outputs.tf [hunk 1]
```diff
@@ -24,3 +26,13 @@ output "app_of_apps_values" {
   sensitive = true
   value     = module.cluster.app_of_apps_values
 }
+
+output "admin_password" {
+  sensitive = true
+  value     = module.cluster.admin_password
+}
```
Comment (mcanevet) [inline examples/eks-aws/terraform/outputs.tf:33]: I'm not sure this is relevant when you are not using Keycloak

**Comment by mcanevet** [comment]
Actually, exposing outputs may not be necessary anymore.
We should remove the dependency on this.

**Comment by mcanevet** [comment]
https://github.com/camptocamp/camptocamp-devops-stack/pull/546 will make this obsolete.

**Comment by fsismondi** [comment]
great! should I close this then?

**Comment by mcanevet** [comment]
yes

**Pull request closed without merging**